health/health_config.c

```diff
@@ -471,6 +471,18 @@ static inline char *health_source_file(size_t line, const char *file) {
     return strdupz(buffer);
 }
 
+static inline char *health_edit_command(size_t line, const char *file) {
+    char buffer[FILENAME_MAX + 1];
+    char *file_no_path = strrchr(file, '/');
+
+    if (likely(file_no_path))
+        snprintfz(buffer, FILENAME_MAX, "sudo %s/edit-config health.d/%s=%zu", netdata_configured_user_config_dir, file_no_path+1, line);
```

Comment:
Hello @MrZammler ,

After the last commit, I decided to test your PR to verify if I was not missing something, I observed that the string is exactly what was planned, as you can see:

```conf
                        "command": "sudo /etc/netdata/edit-config health.d/cpu.conf=4",
                        "command": "sudo /etc/netdata/edit-config health.d/cpu.conf=17",
                        "command": "sudo /etc/netdata/edit-config health.d/cpu.conf=30",
                        "command": "sudo /etc/netdata/edit-config health.d/net.conf=7",
                        "command": "sudo /etc/netdata/edit-config health.d/net.conf=17",
                        "command": "sudo /etc/netdata/edit-config health.d/net.conf=31",
                        "command": "sudo /etc/netdata/edit-config health.d/net.conf=75",
                        "command": "sudo /etc/netdata/edit-config health.d/net.conf=89",
                        "command": "sudo /etc/netdata/edit-config health.d/net.conf=162",
                        "command": "sudo /etc/netdata/edit-config health.d/net.conf=172",
                        "command": "sudo /etc/netdata/edit-config health.d/net.conf=55",
                        "command": "sudo /etc/netdata/edit-config health.d/net.conf=65",
                        "command": "sudo /etc/netdata/edit-config health.d/disks.conf=12",
                        "command": "sudo /etc/netdata/edit-config health.d/disks.conf=26",
                        "command": "sudo /etc/netdata/edit-config health.d/disks.conf=12",
                        "command": "sudo /etc/netdata/edit-config health.d/disks.conf=12",
                        "command": "sudo /etc/netdata/edit-config health.d/disks.conf=26",
                        "command": "sudo /etc/netdata/edit-config health.d/disks.conf=12",
                        "command": "sudo /etc/netdata/edit-config health.d/disks.conf=12",
```

but, if somebody runs this command on terminal, they can have errors depending of default editor, for example, I use `vi` and I had this error:

```bash
bash-5.1$ /etc/netdata/edit-config health.d/disks.conf=12
File 'health.d/disks.conf=12' is not found in '/usr/lib/netdata/conf.d'
```
I propose we add the line as a comment instead part of command line.

Reply:
Hi Thiago, I'm sorry for the confusion, I didn't state it in the original comment.

The string to be sent, contains also the line number. Both the local `alarm-notify.sh` script, and the cloud, when they receive this string, they will split it on the `=` sign. The command then will be printed on the email just up to the point `sudo /etc/netdata/edit-config health.d/disks.conf` and the line number will appear in a different place on the email notification.

Thanks!

Reply:
Interesting, how do you use that line number? And what line is that? 🤔

Reply:
Hi Ilya!

In the `alarm-notify.sh` script, it gets split up (Line 2809 - 2812 in this commit for `alarm-notify.sh` script, sorry can't send a link to that), will split this to the `=` character.

It will then be rendered to the email as:

![mail_edit_command](https://user-images.githubusercontent.com/1905463/112281906-3d349400-8c8f-11eb-9a16-4487a8ff27ae.png)

The line number is already read when reading the conf files, so I didn't mess with that. It is already used in the similar `source` string to emails (which is left too, because it's used in other notification methods).

Reply:
Thanks for the clarification!

Reply:
You are welcome!

Reply:
Thank you for the answer, now it makes more sense.